Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+RUN useradd -m -r ninja && \
+    chown ninja /app
+
+COPY requirements.txt requirements.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY . .
```

Comment:
this line seems to include copied requirements.txt on line 8. you could just get rid of line 8, keep this one and just install after.

Reply:
I was pondering about that. will update